### PR TITLE
Frontier - add GenesisL1

### PR DESCRIPTION
**Pull request by doggystylez**

### CODE REVIEW

**Comment by JeremyParish69** [comment]
Awaiting bech32 prefix registration before merging to chain registry

**Comment by doggystylez** [comment]
sorry, this one isn't done yet. meant to send assetlist first

File: src/config.ts [hunk 1]
```diff
@@ -3607,4 +3613,46 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
...
+			coinDenom: 'L1',
+			coinMinimalDenom: 'el1',
+			coinDecimals: 18,
+			coinGeckoId: 'pool:el1',
+			coinImageUrl: window.location.origin + '/public/assets/tokens/l1.svg',
+		},
+		bip44: {
+			coinType: 118,
+		},
+		bech32Config: Bech32Address.defaultBech32Config('l1'),
```
Comment (JeremyParish69) [inline src/config.ts:3631]: should be 'genesis', not 'l1', as per chain-registry and SLIP-173

**Comment by jonator** [comment]
@doggystylez FYI we've migrated frontier to deploy from the master branch. The main app and frontier share the same codebase except for an env var and 2 different deployments. Please repurpose this PR, or open a PR against master.

**Comment by JeremyParish69** [comment]
Will be replaced by #568

**Pull request closed without merging**